vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -8,13 +8,13 @@ This page provides an overview of the key Entando GitHub repositories with brief
 of how these repositories are realized in a running Entando deployment. It also explores the architecture behind the Entando Platform and how its functions are structured.
 
 ## Entando Operator
-The **Entando Operator** coordinates the installation and configuration of all the components of an Entando Application. 
+The **Entando Operator** coordinates the installation and configuration of all of the components of an Entando Application. 
 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-controller-coordinator/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-controller-coordinator>
 
 #### Customization
-Use the [Entando Custom Resources](../consume/custom-resources.md) to extend the Platform.
+Use the [Entando custom resources](../consume/custom-resources.md) to extend the Platform.
```

Comment:
platform lowercase

Reply:
that actually doesn't make sense because everything else with entando in front remains capitalized

Reply:
no. the platform and operator are the exceptions, because those are general terms. the Component Manager and Component Generator are capitalized because these are entando-specific terms. the decision on platform/operator was made months ago and transferred to our glossary spreadsheet. it may also be in the team notes somewhere.

Reply:
also Entando Component / component, Entando Plugin / plugin, Entando Content / content, Entando Page / page... it's actually a pretty extensive list